hclogslog/adapter.go

```diff
@@ -0,0 +1,204 @@
+package hclogslog
+
+import (
+	"context"
+	"log/slog"
+	"strconv"
+
+	"github.com/hashicorp/go-hclog"
+)
+
+func Adapt(l hclog.Logger) slog.Handler {
+	return &Handler{l: l}
+}
```

Comment:
Should this have a small documentation as entry point for the rest of the adapter?